src/Sylius/Behat/Context/Hook/EmailSpoolContext.php

```diff
@@ -1,38 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace Sylius\Behat\Context\Hook;
-
-use Behat\Behat\Context\Context;
-use Sylius\Component\Core\Test\Services\EmailCheckerInterface;
-use Symfony\Component\Filesystem\Filesystem;
-
-final class EmailSpoolContext implements Context
```

Comment:
I guess it would be nice to add a small note in UPGRADE file as this context is probably used in the suites in end applications

Reply:
It will go in a separate PR o/